File: examples/benchmarks/fib.py
import time

def fib(n):
    if n < 2: return n
    return fib(n-1) + fib(n-2)

start = time.time()
result = fib(32)
end = time.time()

print(f"Fibonacci(32) = {result}")
print(f"Time taken: {end - start:.6f} seconds")

File: examples/benchmarks/fib.lua
function fib(n)
    if n < 2 then return n end
    return fib(n-1) + fib(n-2)
end

local start = os.clock()
local result = fib(32)
local end_time = os.clock()

print("Fibonacci(32) = " .. result)
print("Time taken: " .. (end_time - start) .. " seconds")
